[Bug]: Azure - AttributeError: 'VirtualMachinesOperations' object has no attribute 'power_off'

### Description

Error when powering off Azure machines.

```
        # Stop primary cluster nodes
        if primary_cluster_down:
            config.switch_to_cluster_by_name(primary_cluster_name)
            logger.info(f"Stopping nodes of primary cluster: {primary_cluster_name}")
>           nodes_multicluster[primary_cluster_index].stop_nodes(primary_cluster_nodes)

tests/functional/disaster-recovery/regional-dr/test_failover_and_relocate.py:179: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
ocs_ci/ocs/platform_nodes.py:2386: in stop_nodes
    self.azure.stop_vm_instances(node_names, force=force)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <ocs_ci.utility.azure_utils.AZURE object at 0x7f14944be3d0>
vm_names = ['abhijeet-pd8-b5pbq-master-0', 'abhijeet-pd8-b5pbq-master-1', 'abhijeet-pd8-b5pbq-master-2', 'abhijeet-pd8-b5pbq-worker-eastus1-m2c4x', 'abhijeet-pd8-b5pbq-worker-eastus2-clt2q', 'abhijeet-pd8-b5pbq-worker-eastus3-5msgt', ...]
force = True

    def stop_vm_instances(self, vm_names, force=False):
        """
        Stop Azure vm instances
    
        Args:
            vm_names (list): Names of azure vm instances
            force (bool): True for non-graceful VM shutdown, False for
                graceful VM shutdown
    
        """
        for vm_name in vm_names:
>           result = self.compute_client.virtual_machines.power_off(
                self.cluster_resource_group, vm_name, skip_shutdown=force
            )
E           AttributeError: 'VirtualMachinesOperations' object has no attribute 'power_off'

ocs_ci/utility/azure_utils.py:413: AttributeError
```

### Steps to Reproduce
1. Run the test case tests/functional/disaster-recovery/regional-dr/test_failover_and_relocate.py::TestFailoverAndRelocate::test_failover_and_relocate[primary_down-rbd-cli] or any other node operation terst case on Azure platform.
(after fixing https://github.com/red-hat-storage/ocs-ci/issues/13909)

### Actual Behavior
The method  power_off is not present
### Expected Behavior
Should use the available method to power off
### Impact (likelihood of reproduction, impact on the cluster and on other tests, etc.)
All tests that poweroff the Azure machines

### Screenshots (if applicable)

### Environment
Test Suite(s): RDR (tier4b might be affected)
Platform(s): Azure
Version(s): Tested in ODF, OCP 4.20.
OS:
### Additional Context

--- a/ocs_ci/utility/azure_utils.py
+++ b/ocs_ci/utility/azure_utils.py
@@ -345,7 +345,7 @@ class AZURE:
 
         """
         for vm_name in vm_names:
-            result = self.compute_client.virtual_machines.restart(
+            result = self.compute_client.virtual_machines.begin_restart(
                 self.cluster_resource_group, vm_name
             )
             result.wait()
@@ -394,7 +394,7 @@ class AZURE:
 
         """
         for vm_name in vm_names:
-            result = self.compute_client.virtual_machines.start(
+            result = self.compute_client.virtual_machines.begin_start(
                 self.cluster_resource_group, vm_name
             )
             result.wait()
@@ -410,7 +410,7 @@ class AZURE:
 
         """
         for vm_name in vm_names:
-            result = self.compute_client.virtual_machines.power_off(
+            result = self.compute_client.virtual_machines.begin_power_off(
                 self.cluster_resource_group, vm_name, skip_shutdown=force
             )
             result.wait()

